pkg/graph/resolvers/system_intake_grb_reviewer_test.go

```diff
@@ -0,0 +1,181 @@
+package resolvers
+
+import (
+	"slices"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+
+	"github.com/cmsgov/easi-app/pkg/local"
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/sqlutils"
+	"github.com/cmsgov/easi-app/pkg/userhelpers"
+)
+
+func (s *ResolverSuite) TestSystemIntakeGRBReviewer() {
+	ctx := s.testConfigs.Context
+	store := s.testConfigs.Store
+	okta := local.NewOktaAPIClient()
+
+	s.Run("create GRB Reviewer and add to intake", func() {
+		reviewerEUA := "ABCD"
+		votingRole := models.SystemIntakeGRBReviewerVotingRoleVoting
+		grbRole := models.SystemIntakeGRBReviewerRoleAca3021Rep
+
+		intake := s.createNewIntake()
+		userAcct := s.getOrCreateUserAcct(reviewerEUA)
+
+		reviewer, err := CreateSystemIntakeGRBReviewer(
+			ctx,
+			store,
+			userhelpers.GetUserInfoAccountInfoWrapperFunc(okta.FetchUserInfo),
+			&models.CreateSystemIntakeGRBReviewerInput{
+				SystemIntakeID: intake.ID,
+				EuaUserID:      reviewerEUA,
+				VotingRole:     votingRole,
+				GrbRole:        grbRole,
+			},
+		)
+		s.NoError(err)
+		s.Equal(userAcct.ID, reviewer.UserID)
+		s.Equal(string(votingRole), string(reviewer.VotingRole))
+		s.Equal(string(grbRole), string(reviewer.GRBRole))
+		s.Equal(intake.ID, reviewer.SystemIntakeID)
+	})
+
+	s.Run("fetch GRB reviewers", func() {
+		intake, reviewers := s.createIntakeAndAddReviewers(
+			reviewerToAdd{
+				euaUserID:  "ABCD",
+				votingRole: models.SIGRBRVRVoting,
+				grbRole:    models.SIGRBRRACA3021Rep,
+			},
+			reviewerToAdd{
+				euaUserID:  "TEST",
+				votingRole: models.SIGRBRVRVoting,
+				grbRole:    models.SIGRBRRACA3021Rep,
+			},
+			reviewerToAdd{
+				euaUserID:  "A11Y",
+				votingRole: models.SIGRBRVRVoting,
+				grbRole:    models.SIGRBRRACA3021Rep,
+			},
+		)
+		s.Len(reviewers, 3)
+		fetchedReviewers, err := SystemIntakeGRBReviewers(ctx, intake.ID)
```

Comment:
there is a new resolver suite method to get a fresh set of dataloaders - i can see you are only using the loader once, but in the event you add another test, i think it's best to use this helper

```suggestion
		fetchedReviewers, err := SystemIntakeGRBReviewers(s.ctxWithNewDataloaders(), intake.ID)
```

Reply:
Added in [51e7876](https://github.com/CMSgov/easi-app/pull/2646/commits/51e7876c2fd0bcfdf34a409bda506d31cacfc721)